tutorial.md

```diff
@@ -0,0 +1,234 @@
+$ cd dynamico-demo
+& yarn add @dynamico/core @dynamico/react
+```
+
+Open `App.js` and add these import statements to the top of the file:
+```javascript
+import { DynamicoClient } from '@dynamico/core';
+import { DynamicoProvider } from '@dynamico/react';
+```
+
+Here we import both dynamico client which is the client side 
+library responsible for extracting components from the code 
+and dynamico provider which is used to pass the client along on the context to so you can use dynamic components just like a regular react component.
+
+We're also going to need ReactDOM available in this scope so let's go ahead and import that too:
+```javascript
+import * as ReactDOM from 'react-dom';
+```
+
+Now we're going to initialize the client. It requires a few mandatory parameters. The first is the URL of the registry to be used in production. You can use the demo registry we've set up. the url is: `https://dynamico-demo-registry.soluto.io`.
+> Note: Components you publish to this registry are only available for a day, and are deleted a day after they're published.
+
+The next is the storage to use for local caching of components. The API for the cache matches localStorage's API so you can just pass it as the cache.
+
+Last is the dependencies we provide from the host app to components. We'll pass in just React and ReactDOM for now.
+The dependencies object needs to get the versions of our dependencies and the resolvers for the actual objects. For versions we're just going to pass in our package.json file:
+```javascript
+import { dependencies } from '../package.json';
+```
+
+For the resolvers we'll define this object:
+```javascript
+const resolvers = {
+  react: React,
+  'react-dom': ReactDOM
+};
+```
+
+And now we have everything we need to initialize the client. Add these lines to your code:
+```javascript
+const dynamicoClient = new DynamicoClient({
+  url: 'https://dynamico-demo-registry.soluto.io',
+  dependencies: {
+    versions: dependencies,
+    resolvers
+  },
+  cache: localStorage
+});
+```
+
+Now we're going to initialize our react provider. Doing it is as easy as adding a single component to wrap the App's top level component:
+```javascript
+function App() {
+  return (
+      <DynamicoProvider client={dynamicoClient}>
+        <div className="App">
+        <header className="App-header">
+            <img src={logo} className="App-logo" alt="logo" />
+            <p>
+            Edit <code>src/App.js</code>...
+            </p>
+            <a
+            className="App-link"
+            href="https://reactjs.org"
+            target="_blank"
+            rel="noopener noreferrer"
+            >
+            Learn React
+            </a>
+        </header>
+        </div>
+    </DynamicoProvider>
+  );
+}
+```
+
+Now we can try and run the app:
+```bash
+$ yarn start
+```
+
+And you'll see CRA's default view.
+
+Let's add a dynamic component. Open App.js again and add dynamico to the import statement from @dynamico/react:
```

Comment:
I'd say, let's create a new file

Reply:
I want to keep it simple and a minimal number of steps. Creating a new file requires more work and things to get wrong... I know adding a file and importing it is simple but this is a tutorial, not best practices guide. Am I making sense?